xlml/utils/xpk.py

```diff
@@ -79,17 +71,20 @@ def run_workload(
       ),
   )
 
-  return KubernetesPodOperator(
-      task_id=task_id,
-      name=benchmark_id,
-      cmds=["/bin/bash", "-c"],
-      arguments=[";".join(cmds)],
+  subprocess.run(["bash", "-c", ";".join(cmds)], check=True)
+
+
+# To support local execution using `scripts/local-airflow.sh`, run using
+# task.docker in local environments. Otherwise, task.kubernetes should be used.
+if "XLMLTEST_LOCAL_AIRFLOW" in os.environ:
+  run_workload = task.docker(_run_workload, image="python:3.10")
+else:
+  run_workload = task.kubernetes(
+      _run_workload,
       namespace="composer-user-workloads",
       image="python:3.10",
       config_file="/home/airflow/composer_kube_config",
       kubernetes_conn_id="kubernetes_default",
-      startup_timeout_seconds=startup_timeout,
-      owner=task_owner,
```

Comment:
cc @RissyRan, just want to verify that this is OK to omit. In XPK, workload creation is quick, so `startup_timeout_seconds` doesn't seem necessary. `task_owner` is applied at the TaskGroup level, so I think this is OK to omit. Let me know if I'm missing anything.

Reply:
I recall I met issues to load the image, if that's not the case in multislice anymore, we could remove `startup_timeout_seconds` for now.

for task_owner, discussing offline.

Reply:
I'm able to set `owner` through an override. I'll update the PR. Thanks Ran!